src/python/nnfusion/executor.py

```diff
@@ -80,7 +82,7 @@ class Executor(object):
         5: ("", ""),  # UNKNOWN
     }
 
-    def __init__(self, nnf_rt_dir):
+    def __init__(self, nnf_rt_dir, device=None):
         """
         Parameters:
             nnf_rt_dir: A full string path to nnfusion runtime,
```

Comment:
Could you add docstring for added parameters? And it would be great if there is docstring in runtime.py

Reply:
No problem!